infra/helm/templates/auth-service-service.yaml

```
@@ -3,12 +3,18 @@ kind: Service
   namespace: {{ .Values.namespace }}
 spec:
   ports:
     - name: {{ .Values.authService.port | quote }}
       port: {{ .Values.authService.port }}
       targetPort: {{ .Values.authService.port }}
+    - name: {{ .Values.authService.port | quote }}
+      port: {{ .Values.authService.port }}
+      targetPort: {{ .Values.authService.port }}
   selector:
     io.kompose.service: {{ .Values.authService.name }}
+    io.kompose.service: {{ .Values.authService.name }}
```

YAMLlint (1.37.1)

[error] duplication of key "io.kompose.service" in mapping

(key-duplicates)